src/Akka.Streams.Kafka/Stages/ConsumerStage.cs

```diff
@@ -117,6 +127,21 @@ public override void PostStop()
         private void HandleConsumeError(object sender, Message message)
         {
             Log.Error(message.Error.Reason);
+            var exception = new SerializationException(message.Error.Reason);
+            switch (_decider(exception))
+            {
+                case Directive.Stop:
+                    // Throw
+                    _completion.TrySetException(exception);
+                    FailStage(exception);
+                    break;
+                case Directive.Resume:
+                    // TODO: resume the stream
```

Comment:
@Silv3rcircl3 help me here

Reply:
Don't need to do anything here, just keep going.